terraform/api.tf

```diff
@@ -25,6 +25,7 @@ module "api" {
     django_cors_origin_regex_whitelist = ["^https:\\/\\/[0-9a-z\\-]+\\.netlify\\.app$"]
 
     additional_django_vars = {
+        DANDI_INSTANCE_NAME                            = "LINC"
```

Comment:
Is `"LINC"` the proper value?

Reply:
Yes, I believe so.  We use that instance name in the DANDI CLI: https://github.com/dandi/dandi-cli/blob/03b7c0720329b559431b0872537ded0fc2c13dc9/dandi/consts.py#L135-L139